src/frontends/tensorflow/src/utils.cpp

```diff
@@ -211,3 +214,25 @@ ov::Output<ov::Node> ov::frontend::tensorflow::get_elements_number_1d(const ov::
     auto num_elements = rg.make<Squeeze>(shape);
     return num_elements;
 }
+
+PadMode ov::frontend::tensorflow::convert_padding_mode(const NodeContext& node, const std::string& padding_mode) {
+    std::set<std::string> supported_ops = {"MirrorPad"};
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(
+        node,
+        supported_ops.count(op_type),
+        "OpenVINO TensorFlow Frontend does not support conversion of padding mode for " + op_type + " operation.");
+
+    std::set<std::string> supported_modes = {"REFLECT", "SYMMETRIC"};
+    TENSORFLOW_OP_VALIDATION(node,
+                             supported_modes.count(padding_mode),
+                             "OpenVINO TensorFlow Frontend does not support " + padding_mode + " padding mode.");
+
+    if (padding_mode == "REFLECT") {
+        return PadMode::REFLECT;
+    } else if (padding_mode == "SYMMETRIC") {
+        return PadMode::SYMMETRIC;
+    }
+
+    return PadMode::CONSTANT;
```

Comment:
it looks like this line is not achievable, should we throw exception instead of  `return PadMode::CONSTANT;`?
else {
  // exception
}

Reply:
this is not achievable so we can return any pad mode.

Reply:
I will think of the exception

Reply:
Will fix in the next PR.

Reply:
Fixed.